src/iris/api.py

```diff
@@ -4041,8 +4041,19 @@ def create_email_message(self, application, dest, subject, body):
                 'body': body,
                 'destination': dest
             }
-            message_id = session.execute(sql, data).lastrowid
-            session.commit()
+            retries = 0
+            while retries < 3:
```

Comment:
Can you say a bit about why you're doing this? Do we know this statement to be failing? If so, why? Can we address the underlying problem? Re-trying 3 times with a sleep of 0.2 sec in between seems arbitrary. If you have data to support these values, well & good: document them.
It also seems like the sort of thing for which we should be accumulating metrics, since we'll presumably want to monitor the situation.